pkg/bpfassets/attacher/libbpf_attacher.go

```diff
@@ -331,3 +301,83 @@ func resizeArrayEntries(name string, size int) error {
 
 	return nil
 }
+
+func libbpfCollectProcessBatch(processes *bpf.BPFMap) ([]ProcessBPFMetrics, error) {
+	processesData := []ProcessBPFMetrics{}
+	var err error
+	valueSize := int(unsafe.Sizeof(ProcessBPFMetrics{}))
+	entries := MapSize / valueSize
+	isEnd := false
+	for !isEnd {
+
+		keys := make([]uint32, entries)
+		nextKey := uint32(0)
+
+		val, err := processes.GetValueAndDeleteBatch(unsafe.Pointer(&keys[0]), nil, unsafe.Pointer(&nextKey), uint32(entries))
+		// os.IsNotExist means we reached the end of the table
+		if err != nil {
+			if os.IsNotExist(err) {
+				isEnd = true
+			} else {
+				klog.V(1).Infof("GetValueAndDeleteBatch failed: %v", err)
+				return processesData, err
+			}
+		}
+		processedCount := len(val)
+		if entries != processedCount {
+			isEnd = true
+			klog.V(5).Infof("GetValueAndDeleteBatch request and processed: %d!=%d", entries, processedCount)
+		}
+		for _, value := range val {
+			var ct ProcessBPFMetrics
+			getErr := binary.Read(bytes.NewBuffer(value), ByteOrder, &ct)
+			if getErr != nil {
+				klog.V(1).Infof("failed to decode received data: %v\n", getErr)
+				continue
+			}
+			processesData = append(processesData, ct)
+		}
+	}
+	klog.V(5).Infof("successfully get data with batch get and delete with %d entries", len(processesData))
+	return processesData, err
+}
+func libbpfCollectProcessSingle(processes *bpf.BPFMap) (processesData []ProcessBPFMetrics, err error) {
+	iterator := processes.Iterator()
+	var ct ProcessBPFMetrics
+	keys := []uint32{}
+	retry := 0
+	next := iterator.Next()
+	for next {
```

Comment:
```suggestion
	for it := processes.Iterator(); it.Next() {
```

Reply:
Similarly to above case need another `;` for correct syntax of `for`.

```suggestion
	for it := processes.Iterator(); it.Next(); {
```